stack/setup/main.tf

```diff
@@ -49,18 +49,15 @@ module "secrets-kms-key" {
   deletion_window_in_days = var.kms_deletion_window_in_days
 }
 
-resource "aws_acm_certificate" "main" {
-  domain_name = var.domain
-
-  subject_alternative_names = var.add_wildcard_subdomains ? ["*.${var.domain}"] : []
+module "aws-acm-certificate" {
```

Comment:
This will causes error in setup/cloudflare.tf

Reply:
moved to https://github.com/dbl-works/terraform/pull/91 because I cannot reference a branch that contains a `/`